client/src/fabric/fabric-kernel.ts

```diff
@@ -97,7 +82,7 @@ export class FabricKernel implements IGotchiFactory {
     private spotCenters: Float32Array
     private surface: Int8Array
 
-    constructor(private exports: IFabricExports, dimensions: IFabricDimensions) {
+    constructor(private exports: IFabricExports, private physics: Physics, dimensions: IFabricDimensions) {
```

Comment:
I think the whole "exports" concept needs a better name. Maybe even something WASM-related would be better to distinguish it from CommonJS module exports.

Reply:
Good idea.  This name was adopted from the WASM module stuff, but it could be given a better name, which doesn't conflict.  Maybe just IFabric or IWasmFabric?

Reply:
What exactly is it and what does it do?

Reply:
The interface contains the functions which are exported from the WASM code.

Reply:
What about `Physics`? Is that just an object giving the values for physics features?

Reply:
Physics acquires defaults from the WASM and stores the current values if they are adjusted by the user.

Reply:
Got it. Then `IPhysicsEngine` instead of `IFabricExports`? Maybe `IFabricEngine`? I like putting "engine" in there

Reply:
IFabricEngine sounds good.  I'll add it to the renamings PR.

Reply:
:+1: